server/database/migrations/20190731135151-create-session.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
```

Comment:
Please consider changing this to ES6 exports

Reply:
Done